Makefile.pre.in

```diff
@@ -2245,7 +2245,14 @@ TESTSUBDIRS=	idlelib/idle_test \
 		test/wheeldata \
 		test/xmltestdata \
 		test/xmltestdata/c14n-20 \
-		test/ziptestdata
+		test/ziptestdata \
+		test/support/interpreters \
+		test/test_interpreters \
+		test/test_concurrent_futures \
+		test/test_multiprocessing_fork \
+		test/test_multiprocessing_forkserver \
+		test/test_multiprocessing_spawn \
+		test/test_pathlib
```

Comment:
```suggestion
		test/test_concurrent_futures \
		test/test_multiprocessing_fork \
		test/test_multiprocessing_forkserver \
		test/test_multiprocessing_spawn \
```

test/support/interpreters, test/test_interpreters and test_pathlib don't exist in 3.12 (hence the test failures).